tests/buyers/views/test_buyers.py

```diff
@@ -1874,6 +1887,406 @@ def test_404_if_lot_does_not_allow_brief(self, data_api_client):
+               .assert_called_once_with('view-response-to-requirements')
+
+    def test_get_question_fails_with_empty_list(self):
+        framework_slug = mock.Mock()
+        lot_slug = mock.Mock()
+        manifest = mock.Mock()
+
+        self.content_loader.get_manifest.return_value\
+                           .filter.return_value\
+                           .get_section.return_value = None
+
+        result = self.instance.get_questions(framework_slug, lot_slug, manifest)
+
+        assert result == []
+
+    def test_get_context_data(self):
+        self.instance.get_responses = mock.Mock()
+
+        brief = api_stubs.brief(status='closed')
+
+        kwargs = {
+            'brief_id': mock.Mock(),
+            'framework_slug': mock.Mock(),
+            'lot_slug': mock.Mock()
+        }
+
+        text_type = str if sys.version_info[0] == 3 else unicode
+
+        filename = inflection.parameterize(text_type(brief['briefs']['title']))
+
+        expected = dict(**kwargs)
+        expected['brief'] = brief['briefs']
+        expected['responses'] = self.instance.get_responses.return_value
+        expected['filename'] = 'supplier-responses-' + filename
+
+        self.instance.data_api_client.get_brief.return_value = brief
+
+        with po(buyers, 'get_framework_and_lot') as get_framework_and_lot,\
+                po(buyers, 'is_brief_correct') as is_brief_correct,\
+                mock.patch.object(buyers, 'current_user') as current_user:
+
+            result = self.instance.get_context_data(**kwargs)
+
+        is_brief_correct.assert_called_once_with(brief['briefs'],
+                                                 kwargs['framework_slug'],
+                                                 kwargs['lot_slug'],
+                                                 current_user.id)
+
+        self.instance.data_api_client.get_brief\
+            .assert_called_once_with(kwargs['brief_id'])
+
+        self.instance.get_responses\
+            .assert_called_once_with(brief['briefs'])
+
+        assert result == expected
+
+    def test_get_context_data_with_incorrect_brief(self):
+        self.instance.get_responses = mock.Mock()
+
+        brief = api_stubs.brief(status='closed')
+
+        kwargs = {
+            'brief_id': mock.Mock(),
+            'framework_slug': mock.Mock(),
+            'lot_slug': mock.Mock()
+        }
+
+        self.instance.data_api_client.get_brief.return_value = brief
+
+        with po(buyers, 'get_framework_and_lot') as get_framework_and_lot,\
+                po(buyers, 'is_brief_correct') as is_brief_correct,\
+                mock.patch.object(buyers, 'current_user') as current_user:
+
+            is_brief_correct.return_value = False
+            with pytest.raises(NotFound):
+                self.instance.get_context_data(**kwargs)
+
+    def test_get_context_data_with_open_brief(self):
+        self.instance.get_responses = mock.Mock()
+
+        brief = api_stubs.brief(status='open')
```

Comment:
Should the status be 'live' rather than 'open' - https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models.py#L1197